src/bin/checkrad.in

```diff
@@ -1498,6 +1549,10 @@ if ($ARGV[0] eq 'livingston') {
 	$ret = &dot1x_snmp;
 } elsif ($ARGV[0] eq 'other') {
 	$ret = 1;
+} elsif ($ARGV[0] eq 'juniper_mx') {
+        $ret = &juniper_mx_snmp;
```

Comment:
The diff appears as if we're mixing tabs and spaces

Reply:
I have changed it to tabs